examples/pytorch/pixi.toml

```diff
@@ -0,0 +1,32 @@
+[project]
+channels = []
+name = "pytorch"
+platforms = ["linux-64", "linux-aarch64", "osx-64", "osx-arm64", "win-64"]
+
+[feature.task.tasks]
+start = "python test.py"
+
+[environments]
+cf-channel = ["pytorch-conda-forge", "task"]
+nvidia-channel = ["pytorch-with-cuda-11", "task"]
+pytorch-channel = ["pytorch-from-pytorch", "task"]
+
+[feature.pytorch-conda-forge]
+channels = ["https://fast.prefix.dev/conda-forge"]
+# No windows support YET: https://github.com/conda-forge/pytorch-cpu-feedstock/pull/231
+dependencies = { pytorch = "*" }
+platforms = ["linux-64", "linux-aarch64", "osx-64", "osx-arm64"]
+system-requirements = { cuda = "12.1" }
+
+[feature.pytorch-from-pytorch]
```

Comment:
Examples from pixi mention the use of pytorch from pytorch-channel in conjunction with conda-forge.  However, please note this does not seem supported and can create unstable environments ([ref1](https://github.com/conda-forge/scikit-learn-feedstock/issues/271#issuecomment-2273296035), [ref2](https://github.com/conda-forge/pytorch-cpu-feedstock/issues/249#issuecomment-2273180597)). If you notice well, the pytorch documentation uses `defaults`.  I copied these from the pixi user guide sometime ago and found issues extending the proposed environments with other conda-forge packages (check references). I wonder if the manual should mention unsupported use-cases like this.

Reply:
Thanks for you input! I wasn't aware. I'll take another look. I'm not going to use the defaults channel in one of our example so I might have to wait for the release of the full pytorch work on conda-forge.